### DISCUSSION TITLE
Re running the program will report an error

### DISCUSSION THREAD

**Discussion by YINGCHAO-98** [microsoft/react-native-windows#11029] [Q&A] [answered]
hi
This error will be reported when I close the vs code, reopen the project, and debug Windows.

An error occurred while launching the application. 执行命令 c:\Users\admin\Desktop\react-native-window_dev\interactApp\node_modules\.bin\react-native.cmd run-windows --logging --remote-debugging --no-packager 时出错 (error code 101)
× Building Solution
× Build failed with message C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VisualStudio\v17.0\AppxPackage\Microsoft.AppXPackage.Targets(924,5): error MSB3231: �޷��Ƴ�Ŀ¼��c:\Users\admin\Desktop\react-native-window_dev\interactApp\windows\AppPackages\interactApp\interactApp_1.0.0.0_x64_Debug_Test\����Ŀ¼���ǿյġ� [c:\Users\admin\Desktop\react-native-window_dev\interactApp\windows\interactApp\interactApp.vcxproj]. Check your build configuration.
Please check the 'React Native: Run Windows' output channel for details (error code 1302) (error code 303)

**Comment by chrisglein** [comment, marked as answer]
This looks like a bug report not a discussion. But with bug reports we generally collect all sorts of additional information that's in the issue template. There's not enough information here to help you diagnose. Can you open an issue and include all the information requested for that?
